src/components/ServerGridDashboard/ServerGridDashboard.tsx

```diff
@@ -118,9 +118,7 @@ export class ServerGridDashboard extends React.Component<IServerGridDashboardPro
         this.state = {
             rows: this.transformFarmToRows(props.farms, props.filter),
             expandedRows: {},
-            groupBy: ['FarmName'],
-            sortColumn: 'ServerName',
```

Comment:
The sort configuration is in the Grids state and has no use in ServerGridDashboard